### ISSUE TITLE
install entire freight portal system on Bruce's computer

### ISSUE THREAD

**Issue by tsember** [tsember/freight-portal#39]
- [ ] Use the 'File.separator char' or Path.separator char, whatever

**Comment by jpsember** [comment]
Maybe related to #40.

I just did an experiment:

1. zipped up the repo directory using Finder's 'compress...' feature
1. unzipped it into a temporary directory (outside the repo, to avoid confusion)
1. tried running the jar file; it got some work done before dying:

```
freight-portal> run request
| INFO: ---
| INFO: ---  Logger initialized
| INFO: ---
| INFO: 
| INFO: execute                                                                                                             {RateRequestDesk:LUGCAY}
Exception in thread "main" com.tsember.basic.FileException: java.io.FileNotFoundException: Source '/Users/home/Desktop/experiment/freight-portal/work_template/config-RateRequestDesk.json' does not exist
	at com.tsember.basic.FileTools.asFileException(FileTools.java:19)
	at com.tsember.basic.FileTools.copyFile(FileTools.java:195)
	at com.tsember.Workspace.copyFromWorkTemplateDir(Workspace.java:76)
	at com.tsember.Workspace.file(Workspace.java:50)
	at com.tsember.Workspace.file(Workspace.java:59)
	at com.tsember.app.App.defaultConfigFileLocation(App.java:138)
	at com.tsember.app.App.config(App.java:109)
	at com.tsember.shipmentDesk.RateRequestDesk.initializeDesk(RateRequestDesk.java:42)
	at com.tsember.shipmentDesk.RateRequestDesk.execute(RateRequestDesk.java:29)
	at com.tsember.app.CmdLineApp.main(CmdLineApp.java:45)
Caused by: java.io.FileNotFoundException: Source '/Users/home/Desktop/experiment/freight-portal/work_template/config-RateRequestDesk.json' does not exist
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1074)
	at com.tsember.basic.FileTools.copyFile(FileTools.java:193)
	... 8 more
freight-portal> 
```

**Comment by jpsember** [comment]
Probably because we renamed the class using Eclipse refactoring, which doesn't know about other files: Issue #54

**Comment by jpsember** [comment]
Steps so far:

1. Run `rake zip` to zip the repo up, including the built files, into `repo.zip`
1. Copy `repo.zip` to Bruce's computer
1. Unzip using command line or Windows explorer
1. From the command window, and within the directory just unzipped, run `java -jar target/freight-portal-1.0-jar-with-dependencies.jar`

+ It may complain about backslashes
+ It may not have a 'java' command line program, or it may have an out of date version

**Comment by tsember** [comment]
the issues we encountered:

- Bruce's outlook email strips out the attachements, and it was not obvious to me where it put them.
- gmail does not let you send a zip file.

- we had to rename repo.xyz to repo.zip
- we used windows to extract files
- we had to rename the run scripts to end in .bat

- once we did that, it worked first time.

and here is a test to see if Bruce gets an email if i mention him: Stargate-Freight-Portal
